backend/tools/work_with_db.py

```diff
@@ -0,0 +1,4 @@
+def get_psw_from_db(db, email):
+    log_psw_db = db.AdminInfo.query.filter_by(email=email).first()
```

Comment:
It makes sense to add a paired function that adds a new entry to that table. But it can be done in the next PRs.